src/Server/Provision/ServerProvisioner.php

```diff
@@ -238,11 +238,16 @@ public function getFilters()
         $disks = $this->multiChoice($choices, '/Drive Bay ([0-9]+)(.*)/');
         $addons = $this->addons($this->multiChoice($choices, '/Add On ([0-9]+)/'));
 
+        $configAddons = $this->getConfigValues(WhmcsConfig::ADDON_BILLING_IDS);
+        if ($configAddons) {
```

Comment:
try to avoid updating the contents of a variable as it's confusing - especially if the type of data stored in the variable changes.